### ISSUE TITLE
Question posting error with long title

### ISSUE THREAD

**Issue by jywarren** [publiclab/plots2#1675]
### What happened just before the problem occurred

Someone posted a question with a long title; see below for the error log. We should remove the question title length limitation!!

### Relevant URLs

https://publiclab.org/questions/new

```
Started GET "/questions/new?title=One%20confounding%20issue%20is%20where%20the%20card%20goes%20in%20library%20shipment.%20%20I'm%20looking%20at%20foam-lined%20reusable%20
boxes%20we%20can%20package%20everything%20easily%20and%20minimize%20breakage%20or%20loss." for 144.76.64.79 at 2017-10-11 19:25:42 +0000
Processing by QuestionsController#new as HTML
  Parameters: {"title"=>"One confounding issue is where the card goes in library shipment.  I'm looking at foam-lined reusable boxes we can package everything easily and 
minimize breakage or loss."}
  Rendered editor/questionRich.html.erb within layouts/application (1.3ms)
Completed 500 Internal Server Error in 2.3ms

ActionView::Template::Error (undefined method `id' for nil:NilClass):
    140:         }
    141:       },
    142:       mainImageModule: {
    143:         uid: <%= current_user.id %><% if @node && @node.id %>,
    144:         nid: <%= @node.id %><% end %>
    145:       },
    146:       richTextModule: {
  app/views/editor/questionRich.html.erb:143:in `_app_views_editor_question_ich_html_erb__1645120896819902368_70186117099700'
  app/controllers/questions_controller.rb:34:in `new'
```

**Comment by sagarpreet-chadha** [comment]
@jywarren sir.....can i take this ?

**Comment by jywarren** [comment]
sure, thank you!!!

**Comment by jywarren** [comment]
Fixed in #1688!
